### PR TITLE
Updated docs and tests for `databricks_volume` resource

### PR DESCRIPTION
**Pull request by tanmay-db**
## Changes

Address comments mentioned in: https://github.com/databricks/terraform-provider-databricks/pull/2324

## Tests

- [x] `make test` run locally
- [x] relevant change in `docs/` folder
- [ ] covered with integration tests in `internal/acceptance`
- [ ] relevant acceptance tests are passing
- [ ] using Go SDK

### CODE REVIEW

File: docs/resources/volume.md [hunk 1]
```diff
@@ -29,13 +29,26 @@ resource "databricks_schema" "things" {
   }
 }
 
+resource "databricks_storage_credential" "external" {
+  name = "creds"
+  aws_iam_role {
+    role_arn = "role"
```
Comment (nfx) [inline docs/resources/volume.md:35]: please give more specifcic on what to give into `role_arn` parameter. Preferably, follow the example of other resources and include resource example how to create that role

File: docs/resources/volume.md [hunk 2]
```diff
@@ -29,13 +29,26 @@ resource "databricks_schema" "things" {
...
+  }
+}
+
+resource "databricks_external_location" "some" {
+  name            = "external-location"
+  url             = "some-url"
```
Comment (nfx) [inline docs/resources/volume.md:41]: please be more specific than `some-url`

**Review by nfx — CHANGES_REQUESTED** [review]

File: docs/resources/volume.md (hunk 1 above)
Comment (tanmay-db) [reply docs/resources/volume.md]: Updated.

File: docs/resources/volume.md (hunk 2 above)
Comment (tanmay-db) [reply docs/resources/volume.md]: Updated

**Review by nfx — APPROVED** [review]

**Pull request merged**